testscommon/hashingMocks/hasherMock.go

```diff
@@ -1,4 +1,4 @@
-package mock
```

Comment:
All the functions of the HasherMock are on a HasherMock not on a *HasherMock. Can you please change this?

Reply:
done

Reply:
ok!